docs/architecture.md

```diff
@@ -0,0 +1,63 @@
+# Architecture
+
+- Raw data comes in whatever form
+- They get cleaned up to some common schema
```

Comment:
Worth elaborating that our common schema is count per taxon-location-day? (Or more accurately, that augmented with time in "days after the forecast data)